terraform-provider-constellation/internal/provider/cluster_resource_test.go

```diff
@@ -447,8 +448,8 @@ func TestAccClusterResource(t *testing.T) {
 							kubernetes_version = "%s"
 							constellation_microservice_version = "%s"
 					  }
-				`, versions.Default, providerVersion.String()),
-					ExpectError: regexp.MustCompile(".*When csp is set to 'gcp', 'ip_cidr_pod' must be set.*"),
+				`, versions.Default, providerVersion),
+					ExpectError: regexp.MustCompile(".*When csp is set to 'gcp', 'ip_cidr_pod' must be set.*|.*Image version \\(v[0-9]+\\.[0-9]+\\.[0-9]+\\) incompatible with provider version*"),
```

Comment:
We only expect the first error here. The latter would be a bug.

Reply:
no its not a bug. I can explain the trickiness tomorrow.

Reply:
I fixed it